app/models/miq_request.rb

```diff
@@ -398,7 +398,7 @@ def execute
       :method_name    => "create_request_tasks",
       :zone           => options.fetch(:miq_zone, my_zone),
       :role           => my_role,
-      :tracking_label => "#{self.class.name.underscore}_#{id}",
+      :tracking_label => "#{self.class.name.underscore}_r#{id}",
```

Comment:
This should be moved into a `tracking_label_id` method like we do for the task.  It can live in the `miq_request.rb` file.